core/network/impl/gossiper_broadcast.hpp

```diff
@@ -29,10 +29,14 @@ namespace kagome::network {
     using PrimaryPropose = consensus::grandpa::PrimaryPropose;
 
    public:
-    GossiperBroadcast(libp2p::Host &host, const PeerList &peer_infos);
+    GossiperBroadcast(libp2p::Host &host);
```

Comment:
add explicit

Reply:
No, it does not need here, because it reference (especially not const), and ctor wouldn't be call.

Reply:
In previous commen I say about ctor of argument.

Reply:
![image](https://user-images.githubusercontent.com/20305118/85053659-88bb8e00-b1a3-11ea-9367-e0a6636d0e62.png)